src/main/resources/liquibase/crop_changelog/15_4_1.xml

```diff
@@ -0,0 +1,190 @@
+        <sql dbms="mysql">
+			UPDATE udflds SET fcode  = "ANCEST_DATA" WHERE fcode = "ANCEST" AND ftable = "ATRIBUTS" AND fldno>1551;
+		</sql>
+	</changeSet>
+
+	<changeSet id="v15.4.1-5" author="corina">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT COUNT(*) FROM projectprop WHERE variable_id=8251 AND project_id IN (SELECT project_id FROM project WHERE `name` LIKE '%Nursery Template%');
+			</sqlCheck>
+		</preConditions>
+		<comment>Remove variable STUDY_BM_CODE from nursery template. IBP-3928</comment>
+		<sql dbms="mysql">
+			DELETE FROM projectprop WHERE variable_id=8251 AND project_id IN (SELECT project_id FROM project WHERE `name` LIKE '%Nursery Template%');
+		</sql>
+	</changeSet>
+
+	<changeSet id="v15.4.1-6" author="corina">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT COUNT(*) FROM cvterm where cvterm_id=4021; 
+			</sqlCheck>
+		</preConditions>
+		<comment>Change method of variables with same property-method-scale triplet. IBP-3800</comment>
+		<sql dbms="mysql">
+			SET FOREIGN_KEY_CHECKS=0;
+			
+			REPLACE INTO `cvterm`(`cvterm_id`,`cv_id`,`name`,`definition`,`dbxref_id`,`is_obsolete`,`is_relationshiptype`) VALUES (4021,1020,'Applied at plot level','Applied at plot level',NULL,0,0);
+
+			UPDATE cvterm_relationship SET object_id=4021 WHERE type_id=1210 AND subject_id IN (8258,8259,8253);
+			
+			SET FOREIGN_KEY_CHECKS=1;
+		</sql>
+	</changeSet>
+
+       <changeSet id="v15.4.1-7" author="corina">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">SELECT count(*) > 1 from cvterm WHERE `name` IN ('EW_DW_kgPlot','EW_FW_gPlot','EW_FW_kgPlot');
+			</sqlCheck>
+		</preConditions>
+		<comment>
+			Apply correction on methods of variables EW_DW_kgPlot, EW_FW_gPlot, EW_FW_kgPlot. IBP-3920
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			SET FOREIGN_KEY_CHECKS=0;
+			
+			UPDATE cvterm c1, cvterm_relationship c2
+			SET c2.object_id=61390,
+			c1.definition=REPLACE(definition,'Field weight','Dry weight')
+			WHERE c1.`name`='EW_DW_kgPlot' AND c1.cvterm_id=51574
+			AND c1.cvterm_id=c2.subject_id
+			AND c2.type_id=1210;
+
+			UPDATE cvterm c1, cvterm_relationship c2
+			SET c2.object_id=61398,
+			c1.definition=REPLACE(definition,'Dry weight','Field weight')
+			WHERE c1.`name`='EW_FW_gPlot' AND c1.cvterm_id=51675
+			AND c1.cvterm_id=c2.subject_id
+			AND c2.type_id=1210;
+			
+			UPDATE cvterm c1, cvterm_relationship c2
+			SET c2.object_id=61398,
+			c1.definition=REPLACE(definition,'Dry weight','Field weight')
+			WHERE c1.`name`='EW_FW_kgPlot' AND c1.cvterm_id=50854
+			AND c1.cvterm_id=c2.subject_id
+			AND c2.type_id=1210;
+
+			SET FOREIGN_KEY_CHECKS=1;
+		</sql>
+	</changeSet>
+
+	<changeSet id="v15.4.1-8" author="corina">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT COUNT(*) FROM projectprop pp, project p where pp.variable_id=8265 and pp.project_id=p.project_id and p.deleted=0; 
```

Comment:
What is the variable (say variable `xyz`) that has the same property-scale-method of FertNTotal_kg_ha variable (id=8265)? To ensure that we have consistent variable state of ontologies, that is we want to phase out this variable, we can do it in two scripts:
1. Update any usage of `FertNTotal_kg_ha` variable (all applicable props table like `projectprop, nd_experimentprop, nd_geolocationprop, stockprop`) to use the dupe variable `xyz`
2. Delete variable from ontology tables (current changeset)

This way, it will not be confusing in the long run as with the current script alone, the variable will end up remaining for some and deleted for other crops/deployments.

Reply:
var 8265 has the same PSM triplet as var 8260 (NFert_kg_ha). Changeset has been corrected to first update any usage of 8265 to 8260, then 8265 is deleted from the ontology.